smtprelay.ini

```
@@ -13,17 +13,17 @@
 ; unencrypted connections.
 ;listen = 127.0.0.1:25 [::1]:25
 
-; Listen on the following address for Prometheus
-; metrics exposition
-;metrics_listen = :8080
-
 ; STARTTLS and TLS are also supported but need a
 ; SSL certificate and key.
 ;listen = tls://127.0.0.1:465 tls://[::1]:465
 ;listen = starttls://127.0.0.1:587 starttls://[::1]:587
 ;local_cert = smtpd.pem
 ;local_key  = smtpd.key
 
+; Listen on the following address for Prometheus
+; metrics exposition
+;metrics_listen = :8080
+
```

let's add `log_level` in the example config file
```suggestion

; Minimum log level to output
; levels: trace, debug, info, warning, error, fatal and panic.
;log_level = debug
```

levels are taken from here: https://github.com/sirupsen/logrus#level-logging

---

Ah! Forgot about this, thanks

---

Fixed